project/static/css/styles.css

```diff
@@ -1,145 +1,431 @@
+}
+
+h5 {
+    font-size: 1.333em;
+    line-height: 1em;
+    margin: 1em 0;
+    font-weight: 700;
+}
+
+p {
+    font-size: 1em;
+    line-height: 1.5em;
+    margin: 0em 0;
+    font-family: var(--secondary-font);
+    font-weight: 400;
+    color: var(--dark-text-color);
+}
+
+section {
+    margin: 48px 0px;
+}
+
+
+
+/********** Forms **********/
+
+form {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+form .field {
+    display: flex;
+    flex-direction: column;
+    padding: 0 0 12px;
+}
+
+form .field label {
+    margin-bottom: 6px;
+}
+
+form.logout {
+    margin: 0px;
+}
+
+input {
+    background-color: transparent;
+    outline: none;
+    padding: 12px;
+    border: 1px var(--neutral-50) solid;
+}
+
+textarea {
+    border: 1px var(--neutral-50) solid;
+}
+
+textarea:focus, input:focus {
+    outline: var(--accent-color) solid 1px;
+}
+
+.form-title {
+    font-size: 2rem;
+    text-align: center;
+    height: 18px;
+    border-bottom: 24px solid var(--accent-color);
+    margin-bottom: 35px
+}
+
+
+
+/********** Checkbox **********/
+
+.checkbox {
+    --background: var(--light-color);
+    --border: var(--primary-color);
+    --border-hover:var(--accent-color);
+    --border-active: var(--dark-color);
+    --tick: var(--light-color);
+    position: relative;
+}
+
+.checkbox input, .checkbox svg {
+    width: 21px;
+    height: 21px;
+    display: block;
+}
+
+.checkbox input {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    position: relative;
+    outline: none;
+    background: var(--background);
+    border: none;
+    margin: 0;
+    padding: 0;
+    cursor: pointer;
+    border-radius: 4px;
+    transition: box-shadow 0.3s;
+    box-shadow: inset 0 0 0 var(--s, 1px) var(--b, var(--border));
+}
+
+.checkbox input:hover {
+    --s: 2px;
+    --b: var(--border-hover);
+}
+
+.checkbox input:checked {
+    --b: var(--border-active);
+}
+
+.checkbox svg {
+    pointer-events: none;
+    fill: none;
+    stroke-width: 2px;
+    stroke-linecap: round;
+    stroke-linejoin: round;
+    stroke: var(--stroke, var(--border-active));
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 21px;
+    height: 21px;
+    transform: scale(var(--scale, 1)) translateZ(0);
+}
+
+.checkbox.path input:checked {
+    --s: 2px;
+    transition-delay: 0.4s;
+}
+
+.checkbox.path input:checked + svg {
+    --a: 16.1 86.12;
+    --o: 102.22;
+}
+
+.checkbox.path svg {
+    stroke-dasharray: var(--a, 86.12);
+    stroke-dashoffset: var(--o, 86.12);
+    transition: stroke-dasharray 0.6s, stroke-dashoffset 0.6s;
+}
+
+
+
+/********** Flash messages **********/
+
+.flashes {
+    text-align: center;
+    color: var(--accent-color);
+    background-color: var(--primary-color);
+    padding: 1rem;
+}
+
+
+
+/********** Layout **********/
+
+.header {
+    background-color: var(--dark-color);
     color: var(--light-text-color);
     padding: 12px;
-    height: 12px;
+    height: 50px;
```

Comment:
Header class has height: 50px; this causes a small scroll is that made on purpose? or just a small mistake (with 36px scroll disapear)

Reply:
Header size has been increased to give more space to logo. The mistake probably was made in body height. It will be fixed in next commit